charts/stable/vaultwarden/values.yaml

```diff
@@ -47,7 +47,7 @@ ingress:
         - host: bw.domain.tdl
           paths:
             - path: /notifications/hub
-              pathType: Prefix
+              pathType: Exact
```

Comment:
I don't think this would work, because only one suffix of /notifications/hub needs to go to port 80... This would send all suffixes to 80 instead only the root of /notifications/hub